www/views/setup.html

```diff
@@ -236,17 +236,6 @@ <h2><span data-i18n="AutoBackup"></span>:</h2>
 							</div>
 							<div class="row-fluid">
```

Comment:
@gizmocuz , I think this whole diff has to be removed... now an empty line remains

Reply:
Done